src/app/(app)/changepassword.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState }  from 'react'; 
+import { View , TouchableOpacity , Text , Image , StyleSheet , TextInput } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+
+
+const changepassword = () => {
+  const [oldpassword , Setpassword] = useState('');
+  const [newpassword , Setnewpassword] = useState('');
+  const [confirmpassword , Setconfirmpassword] = useState('');
+  const [SecureTextEntry,SetSecureTextEntry] = useState(true);
```

Comment:
- use consistent naming for variables. e.g. 
`const [oldPassword, setOldPassword] = useState('');`

- use proper camelCase for naming variables for readability.
 
Follow these across the PR